packages/attribute-registry-process/src/services/attributeRegistryService.ts

```diff
@@ -119,3 +153,70 @@ export function createVerifiedAttributeLogic({
 
   return toCreateEventAttributeAdded(newVerifiedAttribute);
 }
+
+export function createCertifiedAttributeLogic({
+  attribute,
+  apiCertifiedAttributeSeed,
+  certifier,
+}: {
+  attribute: WithMetadata<Attribute> | undefined;
+  apiCertifiedAttributeSeed: ApiCertifiedAttributeSeed;
+  certifier: string;
+}): CreateEvent<AttributeEvent> {
+  if (attribute) {
+    throw attributeDuplicate(apiCertifiedAttributeSeed.name);
+  }
+
+  const newCertifiedAttribute: Attribute = {
+    id: uuidv4(),
+    kind: attributeKind.certified,
+    name: apiCertifiedAttributeSeed.name,
+    description: apiCertifiedAttributeSeed.description,
+    creationTime: new Date(),
+    code: undefined,
+    origin: certifier,
+  };
+
+  return toCreateEventAttributeAdded(newCertifiedAttribute);
+}
+
+async function getCertifier(tenantId: string): Promise<string> {
+  const tenant = await readModelService.getTenantById(tenantId);
+  if (!tenant) {
+    throw tenantIdNotFound(tenantId);
+  }
+  const certifier = tenant.data.features
+    .filter((f) => f.type === "Certifier")
+    .map((f) => f.certifierId)
+    .find((id) => id.trim().length > 0);
+
+  if (certifier) {
+    return certifier;
+  } else {
+    throw OrganizationIsNotACertifier(tenantId);
+  }
+}
```

Comment:
I suggest this edit in order to make it more readable

```suggestion
async function getCertifierId(tenantId: string): Promise<string> {
  const tenant = await readModelService.getTenantById(tenantId);
  if (!tenant) {
    throw tenantIdNotFound(tenantId);
  }

  const certifier = tenant.data.features
    .filter(({ type }) => type === "Certifier")
    .find(({ certifierId }) => certifierId.trim().length > 0);

  if (certifier) {
    return certifier.certifierId;
  }
  throw OrganizationIsNotACertifier(tenantId);
}
```